scripts/build-win.py

```diff
@@ -99,6 +99,9 @@ def gngen(arch, sio_root, ffmpeg_root, ssl_root, msdk_root, quic_root, scheme, t
         gn_args.append('rtc_use_h264=true')
     if msdk_root or cg_server:
         gn_args.append('rtc_use_h265=true')
+    if fake_audio:
+        gn_args.append('rtc_include_pulse_audio=false')
```

Comment:
On windows there's no such thing of pulse audio. Suggest you don't touch it.

Reply:
Removed.